### PR TITLE
Bug where vq loss is small compared to recon loss

### PR DESCRIPTION
**Pull request by ctr26**
https://github.com/clementchadebec/benchmark_VAE/issues/124

Attempts to resolve this issue

tl;dr - VQVAE loss needs to be L2 not MSE

### CODE REVIEW

File: src/pythae/models/vq_vae/vq_vae_utils.py [hunk 1]
```diff
@@ -50,14 +50,14 @@ def forward(self, z: torch.Tensor, uses_ddp: bool = False):
...
             z.reshape(-1, self.embedding_dim),
-            reduction="mean",
+            reduction="sum",
         )
 
         embedding_loss = F.mse_loss(
             quantized.reshape(-1, self.embedding_dim),
             z.detach().reshape(-1, self.embedding_dim),
-            reduction="mean",
-        ).mean(dim=-1)
```
Comment (ctr26) [inline src/pythae/models/vq_vae/vq_vae_utils.py:60]: @clementchadebec I'm unsure if second mean needs removing, I believe it does?

Comment (clementchadebec) [reply src/pythae/models/vq_vae/vq_vae_utils.py:60]: @ctr26, yes I think it should be removed since the losses are averaged across the batch in `loss_function` method
https://github.com/clementchadebec/benchmark_VAE/blob/18fe279a16e9ba7db81197cb94c1e91d7d795052/src/pythae/models/vq_vae/vq_vae_model.py#L122-L133

**Comment by ctr26** [comment]
Anything you need from me before this gets merged?

**Review by clementchadebec — APPROVED** [review]
Hi @ctr26 thank you very much for catching this bug and sorry for the late reply. Everything looks good for me :)

**Comment by ctr26** [comment]
Happy to help!
